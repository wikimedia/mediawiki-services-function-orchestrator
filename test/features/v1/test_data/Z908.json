{
    "Z1K1":{
        "Z1K1":"Z9",
        "Z9K1":"Z7"
    },
    "Z7K1":{
        "Z1K1":{
            "Z1K1":"Z9",
            "Z9K1":"Z8"
        },
        "Z8K1":{
            "Z1K1":{
                "Z1K1":"Z9",
                "Z9K1":"Z10"
            },
            "Z10K1":{
                "Z1K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z17"
                },
                "Z17K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z1"
                },
                "Z17K2":{
                    "Z1K1":"Z6",
                    "Z6K1":"Z808K1"
                },
                "Z17K3":{
                    "Z1K1":{
                        "Z1K1":"Z9",
                        "Z9K1":"Z12"
                    },
                    "Z12K1":{
                        "Z1K1":{
                            "Z1K1":"Z9",
                            "Z9K1":"Z10"
                        }
                    }
                }
            },
            "Z10K2":{
                "Z1K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z10"
                }
            }
        },
        "Z8K2":{
            "Z1K1":"Z9",
            "Z9K1":"Z1"
        },
        "Z8K3":{
            "Z1K1":{
                "Z1K1":"Z9",
                "Z9K1":"Z10"
            }
        },
        "Z8K4":{
            "Z1K1":{
                "Z1K1": "Z9",
                "Z9K1": "Z10"
            },
            "Z10K1":{
                "Z1K1": {
                    "Z1K1": "Z9",
                    "Z9K1": "Z14"
                },
                "Z14K1": {
                    "Z1K1":"Z9",
                    "Z9K1":"Z808"
                },
                "Z14K4": {
                    "Z1K1": "Z6",
                    "Z6K1": "Z908"
                }
            }
        },
        "Z8K5":{
            "Z1K1":"Z9",
            "Z9K1":"Z808"
        }
    },
    "Z808K1":{
        "Z1K1":{
            "Z1K1":"Z9",
            "Z9K1":"Z10"
        },
        "Z10K1":{
            "Z1K1":{
                "Z1K1":"Z9",
                "Z9K1":"Z22"
            },
            "Z22K1":{
                "Z1K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z39"
                },
                "Z39K1":{
                    "Z1K1":"Z6",
                    "Z6K1":"Z1K1"
                }
            },
            "Z22K2":{
                "Z1K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z10"
                },
                "Z10K1":{
                    "Z1K1":{
                        "Z1K1":"Z9",
                        "Z9K1":"Z22"
                    },
                    "Z22K1":{
                        "Z1K1":{
                            "Z1K1":"Z9",
                            "Z9K1":"Z39"
                        },
                        "Z39K1":{
                            "Z1K1":"Z6",
                            "Z6K1":"Z1K1"
                        }
                    },
                    "Z22K2":{
                        "Z1K1":"Z6",
                        "Z6K1":"Z9"
                    }
                },
                "Z10K2":{
                    "Z1K1":{
                        "Z1K1":"Z9",
                        "Z9K1":"Z10"
                    },
                    "Z10K1":{
                        "Z1K1":{
                            "Z1K1":"Z9",
                            "Z9K1":"Z22"
                        },
                        "Z22K1":{
                            "Z1K1":{
                                "Z1K1":"Z9",
                                "Z9K1":"Z39"
                            },
                            "Z39K1":{
                                "Z1K1":"Z6",
                                "Z6K1":"Z9K1"
                            }
                        },
                        "Z22K2":{
                            "Z1K1":"Z6",
                            "Z6K1":"Z60"
                        }
                    },
                    "Z10K2":{
                        "Z1K1":{
                            "Z1K1":"Z9",
                            "Z9K1":"Z10"
                        }
                    }
                }
            }
        },
        "Z10K2":{
            "Z1K1":{
                "Z1K1":"Z9",
                "Z9K1":"Z10"
            },
            "Z10K1":{
                "Z1K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z22"
                },
                "Z22K1":{
                    "Z1K1":{
                        "Z1K1":"Z9",
                        "Z9K1":"Z39"
                    },
                    "Z39K1":{
                        "Z1K1":"Z6",
                        "Z6K1":"Z60K1"
                    }
                },
                "Z22K2":{
                    "Z1K1":{
                        "Z1K1":"Z9",
                        "Z9K1":"Z10"
                    },
                    "Z10K1":{
                        "Z1K1":{
                            "Z1K1":"Z9",
                            "Z9K1":"Z22"
                        },
                        "Z22K1":{
                            "Z1K1":{
                                "Z1K1":"Z9",
                                "Z9K1":"Z39"
                            },
                            "Z39K1":{
                                "Z1K1":"Z6",
                                "Z6K1":"Z1K1"
                            }
                        },
                        "Z22K2":{
                            "Z1K1":"Z6",
                            "Z6K1":"Z6"
                        }
                    },
                    "Z10K2":{
                        "Z1K1":{
                            "Z1K1":"Z9",
                            "Z9K1":"Z10"
                        },
                        "Z10K1":{
                            "Z1K1":{
                                "Z1K1":"Z9",
                                "Z9K1":"Z22"
                            },
                            "Z22K1":{
                                "Z1K1":{
                                    "Z1K1":"Z9",
                                    "Z9K1":"Z39"
                                },
                                "Z39K1":{
                                    "Z1K1":"Z6",
                                    "Z6K1":"Z6K1"
                                }
                            },
                            "Z22K2":{
                                "Z1K1":"Z6",
                                "Z6K1":"tlh"
                            }
                        },
                        "Z10K2":{
                            "Z1K1":{
                                "Z1K1":"Z9",
                                "Z9K1":"Z10"
                            }
                        }
                    }
                }
            },
            "Z10K2":{
                "Z1K1":{
                    "Z1K1":"Z9",
                    "Z9K1":"Z10"
                }
            }
        }
    }
}