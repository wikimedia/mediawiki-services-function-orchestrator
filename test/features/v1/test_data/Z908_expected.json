{
    "Z1K1": {"Z1K1": "Z9", "Z9K1": "Z60"},
    "Z60K1": {"Z1K1": "Z6", "Z6K1": "tlh"}
}
